.pipelines/submodules-pipeline.yaml

```diff
@@ -244,32 +245,80 @@ stages:
     parameters:
       name: "cilium_e2e"
       displayName: Cilium
-      pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: true
-      clusterName: "submodules-ciliume2e"
+      clusterType: cilium-podsubnet-up
+      clusterName: "submodciliume2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: 'containerize'
 
   - template: singletenancy/cilium-overlay/cilium-overlay-e2e-job-template.yaml
     parameters:
       name: "cilium_overlay_e2e"
       displayName: Cilium on AKS Overlay
-      pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: true
-      clusterName: "submodules-overlaye2e"
+      clusterType: cilium-overlay-up
+      clusterName: "submodoverlaye2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: 'containerize'
 
   - template: singletenancy/aks-swift/e2e-job-template.yaml
     parameters:
       name: "aks_swift_e2e"
       displayName: AKS Swift
-      pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: true
+      clusterType: swift-byocni-up
       clusterName: "submodswift"
-      osSku: "Ubuntu"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: 'containerize'
+
+  - stage: delete
+    displayName: Delete Clusters
+    condition: always()
+    dependsOn:
+      - setup
+      - "aks_swift_e2e"
+      - "cilium_e2e"
+      - "cilium_overlay_cilium_e2e"
+    variables:
+      commitID: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.commitID'] ]
+    jobs:
+      - job: delete
+        displayName: Delete Cluster
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        strategy:
+          matrix:
+            cilium_e2e:
+              name: cilium_e2e
+              clusterName: 'submodciliume2e'
+              region: $(REGION_AKS_CLUSTER_TEST)
+            cilium_overlay_cilium_e2e:
+              name: cilium_overlay_cilium_e2e
+              clusterName: 'submodoverlaye2e'
+              region: $(REGION_OVERLAY_CLUSTER_TEST)
```

Comment:
What's the default value for the `REGION_AKS_CLUSTER_TEST` and `REGION_OVERLAY_CLUSTER_TEST` ? we are using different regions due to quota reason ?

Reply:
Originally I thought that Overlay was something that may be region specific on our subscription. Since that is not the case, quota for `REGION_OVERLAY_CLUSTER_TEST : westcentralus` is actually quite low for everything. I will change this to default to `REGION_AKS_CLUSTER_TEST : eastus`